tests/integration/test_using_cached_dependencies.py

```diff
@@ -458,9 +457,7 @@ def update_main_repo(env_data, repo_dir, tmpdir, new_dep_commits, dep_repo):
         dep_version = get_pseudo_version(dep_repo, new_dep_commits[0])
 
         with open(os.path.join(repo_dir, "go.mod"), "r") as f:
-            base_gomod_file = [
-                line for line in f.read().split("\n") if not line.startswith("require")
-            ]
+            base_gomod_file = [line for line in f.read().split("\n")]
         with open(os.path.join(repo_dir, "go.mod"), "w+") as f:
             for line in base_gomod_file:
                 f.write(f"{line}\n")
```

Comment:
If you want to simplify this, we just need to open the file once for appending and write a single line to it

Reply:
FTFY